signbank/dictionary/update.py

```diff
@@ -141,7 +141,7 @@ def update_gloss(request, glossid):
                 return HttpResponseBadRequest(_("Unknown field"), content_type='text/plain')
 
             # Translate the value if a boolean
-            if isinstance(Gloss._meta.get_field(field), NullBooleanField):
+            if isinstance(Gloss._meta.get_field(field), NullBooleanField) or (field == 'fingerspelling'):
```

Comment:
```suggestion
            if isinstance(Gloss._meta.get_field(field), BooleanField):
```

[`NullBooleanField` inherits from `BooleanField`](https://github.com/django/django/blob/0dd29209091280ccf34e07c9468746c396b7778e/django/forms/fields.py#L816), so you should just be able to check for BooleanField here.